scene/gui/container.cpp

```diff
@@ -141,6 +141,14 @@ void Container::queue_sort() {
 	pending_sort = true;
 }
 
+Control *Container::as_sortable_control(Node *p_node) const {
+	Control *c = Object::cast_to<Control>(p_node);
+	if (!c || !c->is_visible_in_tree() || c->is_set_as_top_level()) {
```

Comment:
```suggestion
Control *Container::as_sortable_control(Node *p_node, bool p_visible) const {
	Control *c = Object::cast_to<Control>(p_node);
	if (!c || (p_visible && !c->is_visible_in_tree()) || c->is_set_as_top_level()) {
```
Like so

Reply:
For Containers like `CenterContainer` where it was inconsistent, it makes sense to me that invisible children shouldn't be sorted at all. So I think they should be unified and don't think it is necessary to match the original.